src/store/lib/filtering.ts

```diff
@@ -148,8 +148,12 @@ function filterOddCost(card: Card) {
   return card.cost != null && card.cost % 2 !== 0;
 }
 
-function filterXCost(xCost: boolean) {
-  return (card: Card) => xCost && card.cost === -2;
+function filterXCost(card: Card) {
+  return card.cost === -2;
+}
+
+function filterNoCost(card: Card) {
+  return card.cost === undefined;
```

Comment:
ArkhamDB sometimes has some (usually temporary until caught) inconsistencies with `null` and `undefined` in the JSON data, we should cover both `null` and `undefined` here to be safe.

```suggestion
  return card.cost == null;
```

Reply:
Sure! 
However, I am kind of a `==` hater. You have to think twice when using it, and it's the [reason why real programmers make fun of javascript](https://www.destroyallsoftware.com/talks/wat).

Reply:
@kidManuel so there is a lint for `===` against `==`, and in recent defaults, it explicitly ignores the case `== null` as it is actually useful.

Reply:
The Wat talk is awesome (and valid), but I really don't think it applies here...